formulas/wick-base/functions/wick-service-add-dependency

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Internal: Makes the service provided depend on another service
+# which could change the order services are loaded in at boot time.
+#
+# $1 - The name of the service you wish to add a dependency to.
+# $2 - The name of the service you wish to add as a dependency to the service.
+#
+# Returns: 1 if $1 is not provided and 2 if $2 is not provided.
+wickServiceAddDependency() {
+    local dependency initType service
+
+    wickExplorer initType wick-base init-system
+    wickGetArgument service 0 "$@"
+    wickGetArgument dependency 1 "$@"
+
+    if [[ -z "$service" ]]; then
+        wickError "wickServiceAddDependency requires a service to be provided as the first argument"
+        return 1
+    fi
+
+    if [[ -z "$dependency" ]]; then
+        wickError "wickServiceAddDependency requires a dependency to be provided as the second argument"
+        return 2
+    fi
+
+    wickServiceMakeOverride "$@"
+
+    wickInfo "Adding dependency \"$dependency\" to service \"$service\""
+    if [[ systemd == "$initType" ]]; then
```

Comment:
Please add newlines before logic blocks (like this "if") to separate it from other blocks and other lines of code.  The only exceptions to this would be if there is a comment before the block or if the block is the first statement in another block.

Also add newlines before `return` to offset them from normal code and hopefully catch people's eyes more.